change/@fluentui-react-tabster-febe226f-524c-446a-a2b1-769b8ba7180f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 32 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26089&commitId=1306708873ca17f5948d290bb288efd9bb4b54d2&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1837 screenshots unchanged, 0 screenshots with different dimensions and 32 screenshots with visible difference.


  <details><summary><b>unknown</b> 32 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link Converged - Rendered as anchor.Inline - RTL.focused.chromium.png|162| Changed|
|Link Converged - Rendered as anchor.Inline - RTL.pressed.chromium.png|190| Changed|
|Link Converged - Rendered as anchor.Inline Disabled Focusable.focused.chromium.png|623| Changed|
|Link Converged - Rendered as anchor.Inline Disabled Focusable.pressed.chromium.png|623| Changed|
|Link Converged - Rendered as anchor.Inline.focused.chromium.png|162| Changed|
|Link Converged - Rendered as anchor.Inline.pressed.chromium.png|190| Changed|
|Link Converged - Rendered as anchor.Stand-alone - Dark Mode.focused.chromium.png|178| Changed|
|Link Converged - Rendered as anchor.Stand-alone - Dark Mode.pressed.chromium.png|94| Changed|
|Link Converged - Rendered as anchor.Stand-alone - High Contrast.focused.chromium.png|140| Changed|
|Link Converged - Rendered as anchor.Stand-alone - High Contrast.pressed.chromium.png|84| Changed|
|Link Converged - Rendered as anchor.Stand-alone - RTL.focused.chromium.png|353| Changed|
|Link Converged - Rendered as anchor.Stand-alone - RTL.pressed.chromium.png|440| Changed|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - Dark Mode.focused.chromium.png|489| Changed|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - Dark Mode.pressed.chromium.png|489| Changed|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - High Contrast.focused.chromium.png|533| Changed|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - High Contrast.pressed.chromium.png|533| Changed|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable.focused.chromium.png|873| Changed|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable.pressed.chromium.png|873| Changed|
|Link Converged - Rendered as anchor.Stand-alone.focused.chromium.png|352| Changed|
|Link Converged - Rendered as anchor.Stand-alone.pressed.chromium.png|440| Changed|
|Link Converged - Rendered as button.Inline - RTL.focused.chromium.png|146| Changed|
|Link Converged - Rendered as button.Inline - RTL.pressed.chromium.png|172| Changed|
|Link Converged - Rendered as button.Inline Disabled Focusable.focused.chromium.png|623| Changed|
|Link Converged - Rendered as button.Inline Disabled Focusable.pressed.chromium.png|623| Changed|
|Link Converged - Rendered as button.Inline.focused.chromium.png|146| Changed|
|Link Converged - Rendered as button.Inline.pressed.chromium.png|172| Changed|
|Link Converged - Rendered as button.Stand-alone - RTL.focused.chromium.png|337| Changed|
|Link Converged - Rendered as button.Stand-alone - RTL.pressed.chromium.png|420| Changed|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable.focused.chromium.png|873| Changed|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable.pressed.chromium.png|873| Changed|
|Link Converged - Rendered as button.Stand-alone.focused.chromium.png|337| Changed|
|Link Converged - Rendered as button.Stand-alone.pressed.chromium.png|420| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>